src/ai-search-app.ts

```
@@ -0,0 +1,141 @@
+      error: error instanceof Error ? error.message : String(error)
+    });
+  }
+}
+
+function startScheduling(schedulingConfig: any, processFunction: () => Promise<void>, name: string) {
+  switch (schedulingConfig.mode) {
+    case 'interval':
+      if (!schedulingConfig.interval?.minutes) {
+        throw new Error(`${name} interval minutes not configured`);
+      }
+      const intervalMs = schedulingConfig.interval.minutes * 60 * 1000;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)